Kickstarter-iOS/Views/Cells/ProjectPamphletSubpageCell.swift

```diff
@@ -94,6 +94,6 @@ internal final class ProjectPamphletSubpageCell: UITableViewCell, ValueCell {
 
   internal override func layoutSubviews() {
     super.layoutSubviews()
-    self.countContainerView.layer.cornerRadius = self.countContainerView.bounds.height / 2
+    self.countContainerView.layer.cornerRadius = min(14.0, round(self.countContainerView.bounds.height / 2.0))
```

Comment:
yes, the rounded corners were more like a diamond. debugger showed that this figure is much higher before the text is set, so I found `14.0` to be the minimum for looking good.